test/ethereum/network/evmScanTxProcessing.test.ts

```diff
@@ -0,0 +1,36 @@
+import { assert } from 'chai'
+import { asEither } from 'cleaners'
+import { describe, it } from 'mocha'
+
+import {
+  asEvmScanInternalTransaction,
+  asEvmScanTransaction
+} from '../../../src/ethereum/ethereumTypes'
+import { processEvmScanTransaction } from '../../../src/ethereum/networkAdapters/EvmScanAdapter'
+import { testCases as processEvmScanTransactionTestCases } from './evmScanTxProcessing.cases'
+
+describe(`processEvmScanTransaction`, function () {
```

Comment:
Can we put the test cases in the same file as the logic? I don't think separating them makes the code more readable.